games/armagetronad/Portfile

```diff
@@ -0,0 +1,291 @@
+    }
+    if {$subport eq "${name}-dedicated"} {
+        # Temporarily relocate the server scripts to a safe location,
+        # since the entire parent directory is about to be deleted
+        # in the code below.
+        move {*}[glob -directory ${destroot}${prefix} \
+             $scriptsdir $rc_config $settings_cfg] \
+             ${destroot}${prefix}/
+    } else {
+        delete {*}[glob -directory ${destroot}${prefix} \
+            $scriptsdir $rc_config $settings_cfg]
+    }
+
+    if {$subport eq ${name}} {
+        # Temporarily relocate desktop-related items to a safe location,
+        # since the entire parent directory is about to be deleted
+        # in the code below.
+        move ${destroot}${prefix}/share/$subport/desktop \
+             ${destroot}${prefix}/
+    }
+
+    if {$subport ne "${name}-common"} {
+        # Delete files that are common to both the client and server
+        delete {*}[glob -directory ${destroot}${prefix} \
+            etc/$subport share/$subport share/doc/$subport]
+    }
+
+    if {$subport eq "${name}-dedicated"} {
+        reinplace -E \
+            "s/\((CONFIG|DATADIR)=.*\)$subport/\\1${name}/" \
+            {*}[glob -directory ${destroot}${prefix}/scripts \
+            rcd_config serverstarter]
+
+        # Restore the server scripts to their original location
+        xinstall -d ${destroot}${prefix}/share/$subport
+        move ${destroot}${prefix}/scripts \
+             ${destroot}${prefix}/share/$subport/
+        xinstall -d ${destroot}${prefix}/etc/${name}
+        move {*}[glob -directory ${destroot}${prefix} \
+             rc.config settings_dedicated.cfg] \
+             ${destroot}${prefix}/etc/${name}/
+    } elseif {$subport eq ${name}} {
+        # Restore the desktop-related items to their original location
+        xinstall -d ${destroot}${prefix}/share/$subport
+        move ${destroot}${prefix}/desktop \
+             ${destroot}${prefix}/share/$subport/
+    }
+
+    if {$subport eq "${name}-common"} {
+        delete {*}[glob -directory ${destroot}${prefix} \
+            bin/${name} share/${name}/desktop \
+            share/${name}/language/update.py]
+    }
+
+    if {$subport ne "${name}-common"} {
+        if {$subport eq "${name}-dedicated"} {
+            foreach re [list \
+                "s|\(/etc\)|${prefix}\\1|" \
+                "s|etc/init.d/$subport|share/$subport/scripts/rcd_server|" \
+                "s|default/$subport|${name}/rc.config|" \
+                "s|/usr|${prefix}|" \
+                "s/${name}-common/${name}/" \
+            ] {
+                reinplace -E $re ${workpath}/debian/man/${subport}.6
+            }
+        } elseif {$subport eq ${name}} {
+            reinplace -E \
+                "s|/usr\(/share/doc/\)${name}-common|${prefix}\\1${name}|" \
+                ${workpath}/debian/man/${subport}.6
+        }
+        system -W ${workpath}/debian/man [join [list \
+            "gzip -9c ${subport}.6" \
```

Comment:
Don't gzip manpage manually; MacPorts will do it for you. If you gzip them, MacPorts will just ungzip them and regzip them.

Reply:
Oh, I didn't know that. I'll just copy the manpage into destroot, then. 👍